### ISSUE TITLE
I can't get the bbox and seg masks in inference_ullava.py

### ISSUE THREAD

**Issue by liuxiang09** [OPPOMKLab/u-LLaVA#6]
Hello author, thank you very much for your work.

But now I have some problems. I downloaded the model weights of Stage 2 from https://huggingface.co/jinxu95/ullava/tree/main and performed inference in inference_ullava.py. I can get the text output correctly, but I **can't get any bbox and segment mask**. The console output is as follows:

=================================================================================================
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Loading checkpoint shards:   0%|                                                                                                                            | 0/2 [00:00<?, ?it/s]/home/hpc/anaconda3/envs/llava-next/lib/python3.10/site-packages/torch/_utils.py:831: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()
  return self.fget.__get__(instance, owner)()
Loading checkpoint shards: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:01<00:00,  1.57it/s]
Some weights of the model checkpoint at ./checkpoints/ullava were not used when initializing UllavaForCausalLM: ['text_hidden_fcs.0.bias', 'text_hidden_fcs.0.weight', 'text_hidden_fcs.2.bias', 'text_hidden_fcs.2.weight']
- This IS expected if you are initializing UllavaForCausalLM from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing UllavaForCausalLM from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Some weights of UllavaForCausalLM were not initialized from the model checkpoint at ./checkpoints/ullava and are newly initialized: ['det_decoder.0.bias', 'det_decoder.0.weight', 'det_decoder.2.bias', 'det_decoder.2.weight', 'det_decoder.4.bias', 'det_decoder.4.weight', 'det_projector.0.bias', 'det_projector.0.weight', 'det_projector.2.bias', 'det_projector.2.weight', 'seg_projector.0.bias', 'seg_projector.0.weight', 'seg_projector.2.bias', 'seg_projector.2.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
/home/hpc/anaconda3/envs/llava-next/lib/python3.10/site-packages/huggingface_hub/file_download.py:943: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
Please input your prompt: Where are the women?
Please input the image path: /mnt/2753047e-bb0d-4a84-9488-1fce437519b3/coco2014/val2014/000000000536.jpg
uLLaVA: Sure. Info: [SEG] ; [tag] person [/tag] .
[ WARN:0@30.657] global loadsave.cpp:848 imwrite_ Unsupported depth image for selected encoder is fallbacked to CV_8U.
./temp/000000000536_mask_0.jpg has been saved.
./temp/000000000536_masked_img_0.jpg has been saved.
./temp/000000000536_bbox_0.jpg has been saved.
=================================================================================================

<img width="824" height="178" alt="Image" src="https://github.com/user-attachments/assets/f12da717-4eca-414a-9e06-5d6883a53547" />

I downloaded the model file locally and loaded CLIPProcessor using `openai/clip-vit-large-patch14`.

Through debugging, I found that the `pred_masks` and `pred_boxes` output by the model are both `nan`. How can I solve this problem? Thank you very much

**Comment by abhigoku10** [comment]
@VeritasXu  any updates on this thread
